assets/images/logo/README.md

```diff
@@ -0,0 +1,4 @@
+# Logo
+CC-BY Liscense
```

Comment:
I’m not sure if you can use this license for a partly copyrighted logo, would be better to have it approved by the team (I’m not even sure you can use it).

You’ve got one superfluous `s` in ‘license’.

Reply:
Sorry for that spelling error, I made the README very quickly.
According to Wikimedia, the logo is too simple to be copyrighted. (You can't copyright the letter V). But (as wikimedia says) you *can* copyright fonts. Let me see if I can get permission just in case.

Reply:
This is (part of the reason) why it's marked as a draft.